app.py

```diff
@@ -262,7 +263,6 @@ def gen(camera):
         yield (b'--frame\r\n' b'Content-Type: image/jpeg\r\n\r\n' + frame + b'\r\n')
 
 def gen_student():
-    time.sleep(15)

```

Comment:
We need this time delay as the user experience should be like a investigation app which does few animations and evaluates the percentage of similarity and then after that shows the matched student face at the end.

Reply:
> Rest LGTM! But approving this pr would delete `main.py`, `style1.css` and `head.html` files. I need those files. Can you please resolve the conflicts? I think after that we can get a better view around it.

I removed the style1.css and head.html file because I changed the way that the zoom in animation would work as well. 

It's likely that we'll need to run this more than once, maybe every time someone new walks into the view of the camera. We wouldn't create a behavior that only happens once otherwise almost everyone who walked by the piece would miss it. 

And since we'll need to run it again it doesn't make sense to tie it to one html page and then load a completely different one. The implementation of using a css class is great. But in this pr I'm proposing that you use Javascript to remove the class when the animation is over. This way you could easily trigger the animation again by having the class be added back by javascript. And you avoid the code duplication of an almost identical html page. This ways a lot simpler. 

As for the main.py file I thought that was only related to running the file as not part of the app. Do you need this?

Reply:
> We need this time delay as the user experience should be like a investigation app which does few animations and evaluates the percentage of similarity and then after that shows the matched student face at the end.

I can add this back

Reply:
but this would only run once. When we implement a functionality like this, pausing between users it would never just run one time. And we don't want to force the page to refresh each time someone new comes into frame. I get what your saying but there's gotta be a better way to implement it.

Reply:
Personally I'd remove it and see where the app is at then reevaluate how to achieve this.

Reply:
Stephan asked that there should be no delay. So, we can remove it altogether.